sound/soc/sof/intel/hda.h

```diff
@@ -918,6 +918,7 @@ int hda_dsp_ipc4_load_library(struct snd_sof_dev *sdev,
  * @codec_dai_set_hext_stream: Function pointer to set codec-side stream information
  * @calc_stream_format: Function pointer to determine stream format from hw_params and
  * possibly codec DAI .sig bits
+ * @get_hlink: Mandatory function pointer to retrieve hlink, mainly to program LOSIDV
```

Comment:
the comment makes no sense, I''ll update this @ujfalusi . Buf so that it's clearer, here's what the callback would look like for SSP:

````c
static struct hdac_ext_link *ssp_get_hlink(struct snd_sof_dev *sdev,
					   struct snd_pcm_substream *substream)
{
	struct hdac_bus *bus = sof_to_bus(sdev);

	return hdac_bus_eml_ssp_get_hlink(bus);
}
````